assets/ajax-cart.js.liquid

```diff
@@ -354,6 +386,7 @@ var ajaxCart = (function(module, $) {
 
       // If it has a data-line, update the cart.
       // Otherwise, just update the input's number
+      $body.trigger('adjustCart.ajaxCart', [this, line, qty]);
```

Comment:
Can remove this. `updateQuantity` (the function right after this) calls `ShopifyAPI.changeItem`, which already has triggers fired (`before/after/error/complete ChangeItem`)